src/agent/onefuzz-agent/src/worker.rs

```diff
@@ -193,10 +197,24 @@ impl State<Ready> {
         .await
         {
             Err(e) => {
-                error!("timeout waiting for client_sender_server.accept(): {:?}", e);
+                let _: Elapsed = e; // error here is always Elapsed and has no further info
+
+                // see if child exited with any useful information:
+                let child_output = match child.try_wait() {
+                    Ok(None) => "still running".to_string(),
+                    Ok(Some(output)) => {
+                        format!("{:?}", output)
+                    }
+                    Err(e) => format!("{}", e),
+                };
+
+                error!(
+                    "timeout waiting for client_sender_server.accept(): child status: {}",
+                    child_output,
+                );
+
                 return Err(format_err!(
-                    "timeout waiting for client_sender_server.accept(): {:?}",
-                    e
+                    "timeout waiting for client_sender_server.accept()"
```

Comment:
we need include the child output here so it will appear in the error field of the task